src/handlers/parseRawDependencyDataHandler.ts

```diff
@@ -20,50 +20,51 @@ export async function parseRawDependencyDataHandler(project: MavenProject): Prom
     const re = /([\w.]+:[\w.-]+:)([\w.-]+)(:[\w/.(\s]+):\s([\w.-]+)\)/gm;
     treeContent = treeContent.replace(re, "$1$4$3 with $2)");
 
-    const eol: string = "\r\n";
     const indent: string = "   "; // three spaces
     const separator: string = "\r\n";
     const starter: string = "+- ";
-    const treeNodes: Dependency[] = parseTreeNodes(treeContent, separator, indent, starter, eol, project.pomPath);
-    setOldestAncestor(treeNodes);
-    return treeNodes;
+    return parseTreeNodes(treeContent, separator, indent, starter, project.pomPath);
 }
 
-function parseTreeNodes(treecontent: string, separator: string, indent: string, starter: string, eol: string, projectPomPath: string): Dependency[] {
+function parseTreeNodes(treecontent: string, separator: string, indent: string, starter: string, projectPomPath: string): Dependency[] {
     const treeNodes: Dependency[] = [];
     if (treecontent) {
-        const treeChildren: string[] = treecontent.split(`${separator}${starter}`).splice(1); // delete first line
-        const toTreeNode = (treeChild: string): Dependency => {
-            let curNode: Dependency;
-            if (treeChild.indexOf(eol) === -1) {
-                curNode = new Dependency(treeChild, projectPomPath);
+        let curNode: Dependency;
+        let preNode: Dependency;
+        let parentNode: Dependency;
+        let rootNode: Dependency;
+        let curIndex: number;
+        let preIndex: number = 1;
+        const treeChildren: string[] = treecontent.split(separator).splice(1); // delete first line
```

Comment:
> // delete first line

everyone knows you are deleting the first line (if you rename separator to eol). Key point is why are you doing this.

Reply:
rename `treeChildren` to `lines`